README.md

```diff
@@ -152,26 +152,37 @@ kubectl get nodes
 
 ## Local development
 
-This Quick Start is developed as a set of AWS CloudFormation templates.  This is a brief overview of the files in this repo, for more architecture details see the [Deployment Guide](https://s3.amazonaws.com/quickstart-reference/heptio/latest/doc/heptio-kubernetes-on-the-aws-cloud.pdf)
+This Quick Start is developed as a set of AWS CloudFormation templates. This is
+a brief overview of the files in this repo, for more architecture details see
+the [Deployment
+Guide](https://s3.amazonaws.com/quickstart-reference/heptio/latest/doc/heptio-kubernetes-on-the-aws-cloud.pdf)
 
 **The `templates` directory**:
 
-- `kubernetes-cluster.template` sets up the resources that the Kubernetes cluster depends on.  Mainly this is the API load balancer, the master node, the auto-scaling group of kubelet nodes, and the various security groups required to allow them to talk to one another.  The nodes in this template are created from a base AMI, which you can recreate using the scripts in the `packer` directory in this repo.
-- `kubernetes-cluster-with-new-vpc.template` sets up a new VPC with a public and private subnets, and calls out to `kubernetes-cluster.template` as a sub-stack.
+- `kubernetes-cluster.template` sets up the resources that the Kubernetes
+  cluster depends on. Mainly this is the API load balancer, the master node, the
+  auto-scaling group of kubelet nodes, and the various security groups required
+  to allow them to talk to one another. The nodes in this template are created
+  from a base AMI, which you can recreate using
+  [wardroom](https://github.com/heptiolabs/wardroom).
+- `kubernetes-cluster-with-new-vpc.template` sets up a new VPC with a public and
+  private subnets, and calls out to `kubernetes-cluster.template` as a
+  sub-stack.
 
 **The `scripts` directory**:
 
-This contains files that required by the templates.  This is to avoid inlining a lot of text directly in the template files.  Templates access files from this directory by referencing their S3 URL's, which is why this directory and the `templates` directory both need to be copied into S3 in order to be used.
+This contains files that required by the templates. This is to avoid inlining a
```

Comment:
should be

> that are required

(although I recognize you didn't write this, so if it makes sense to open a separate ticket for proof reading existing documentation that's fine by me)